### DISCUSSION TITLE
Welcome to salvo Discussions!

### DISCUSSION THREAD

**Discussion by chrislearn** [salvo-rs/salvo#19] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by saiumesh535** [comment]
Hello There! I just came across this framework and loved how simple it is to write API(s) especially the middleware part. Having said that, I think we can add a few features like the following. 

1. Serving static files 
2. Cors

If these are not planned already? I can try working on one of them. 

Thank you so much for this framework. looks great.

**Reply by chrislearn** [reply]
Very glad you like salvo. 
1. Serving static files: https://github.com/salvo-rs/salvo/tree/master/extra/src/serve   example: https://github.com/salvo-rs/salvo/blob/master/examples/file_list.rs
2. Cors: https://github.com/salvo-rs/salvo/tree/master/extra/src/cors

**Reply by saiumesh535** [reply]
Hey, @chrislearn Thanks, for the update. Looks good!
